mbedtls/src/ecp/mod.rs

```diff
@@ -335,8 +334,8 @@ impl EcPoint {
                 &mut ret.inner,
                 k.handle(),
                 &self.inner,
-                None,
-                ::core::ptr::null_mut(),
+                Some(F::call),
```

Comment:
this change is important security-wise; as we were not blinding before... perhaps highlight it in the changelog.